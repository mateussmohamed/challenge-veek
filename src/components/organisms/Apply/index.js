import React from 'react'
import styled from 'styled-components'

import { Grid, Row, Col } from 'react-styled-flexboxgrid'

import { Typography } from 'atoms'
import { FormApply } from 'organisms'

const ApplyContainer = styled.div`
  width: 100%;
`

const Apply = () => (
  <Grid>
    <ApplyContainer>
      <Row style={{ marginBottom: 50 }}>
        <Col xs={12}>
          <Typography variant="title2">Apply</Typography>
          <Typography variant="subtitle2">
            Thought experiments (Gedankenexperimenten) are “facts” in the sense{' '}
          </Typography>
        </Col>
      </Row>
      <FormApply />
    </ApplyContainer>
  </Grid>
)

export default Apply
