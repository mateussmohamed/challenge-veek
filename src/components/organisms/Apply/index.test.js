import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Apply from './index'

describe('[<Apply />]', () => {
  const wrap = (props = {}) => shallow(<Apply {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
