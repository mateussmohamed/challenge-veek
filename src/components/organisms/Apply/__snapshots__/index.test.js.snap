// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Apply />] matches to snapshot 1`] = `
<Grid>
  <Apply__ApplyContainer>
    <Row
      style={
        Object {
          "marginBottom": 50,
        }
      }
    >
      <Col
        xs={12}
      >
        <Typography
          variant="title2"
        >
          Apply
        </Typography>
        <Typography
          variant="subtitle2"
        >
          Thought experiments (Gedankenexperimenten) are “facts” in the sense
           
        </Typography>
      </Col>
    </Row>
    <FormApply />
  </Apply__ApplyContainer>
</Grid>
`;
